R/check_result.R

```diff
@@ -16,6 +16,7 @@
 #' @template learnr_args
 #' @param ... ignored
 #'
+#' TODO eventuall will return a grader_condition object from condi
```

Comment:
returning a `grader_graded` is correct.  While it may contain similar info, the mental model is different and should be kept separate.

* A `condi` is a condition used to determine a `graded` value